docs/thehive/release-notes/release-notes-5.0.md

```diff
@@ -0,0 +1,43 @@
+# Release Notes of 5.0 series
+
+## 5.0.1 - 7 April 2022
+
+TheHive 5.0.1 is the first patch release in the 5.0 series. It contains fixes and improvements for the UI and some small changes for the API compared to 5.0.0. 
+
+We recommend all 5.0.0 users to update to this version.
+
+### Notables changes
+
+**UI:**
+
+- fix description display on search page
+- display org admin tabs only with required permissions
+- fix permission for knowledge base
```

Comment:
permissions have been fixed globally (a lot of entities were'nt checked properly like responders, tasks, task logs etc...), maybe write:
` - fix permissions check in the entire app`

Reply:
So that there is no confusion about the role of the front-end on the security of the app :
``- add more consistency between the interface and the authorizations ``